src/functionalTest/resources/features/F-701 - createCasePaymentOrder/F-701.feature

```diff
@@ -0,0 +1,95 @@
+@F-701
+Feature: CREATE Case Payment Order Endpoint
+
+
+  Background:
+    Given an appropriate test context as detailed in the test data source,
+    Given a user with [an active profile in CCD],
+
+  @S-701.1
+  Scenario: Successfully allow the creation of a case payment order in the Case Payment Order database
+
+    When a request is prepared with appropriate values
+    And the request [intends to Create a case payment order]
+    And the request [contains all the mandatory parameters]
+    And it is submitted to call the [createCasePaymentOrder] operation of [Case Payment Orders Microservice],
+    Then a positive response is received
+    And the response [contains a 200 success OK code]
+    And the response [contains a valid UUID of the case order record]
+    And the response has all other details as expected,
+    And a call [to verify that a Case payment Order has been created] will get the expected response as in [getCasePaymentOrder_cpo_created].
+
+
+
+  Scenario Outline: AC2- Must return error if one or more of the mandatory parameters have not been provided (Please refer to the mandatory parameter list in the description)
+    When a request is prepared with appropriate values
+    And the request [intends to Create a case payment order]
+    And the request [does not contain one or more of the mandatory parameters]
+    And the request [does not contain the <parameter> parameter]
+    And it is submitted to call the [createCasePaymentOrder] operation of [Case Payment Orders Microservice],
+    Then a negative response is received
+    And the response has all other details as expected
+    And a call [to verify that a Case payment Order has not been created in the database] will get the expected response as in [getCasePaymentOrder_cpo_not_created].
+
+  @Ignore #@S-701.2.1 TODO re-enable after CPO-40 is fixed
```

Comment:
CPO-40 is marked as "Withdrawn" in Jira ?

Reply:
correct, can remove this scenario now